src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -135,18 +141,40 @@ export function NavGroups({
     }
 
     if (navLink.itemType === LinkItemType.PARENT_LINK && navLink.link) {
-      return {
-        ...createNavItem({ link: navLink.link }),
+      const props = createNavItem({ link: navLink.link });
+      const parentItem = {
+        ...props,
         forceOpen: true,
-        items: navLink.links.map((subNavLink) => createSideNavItem(subNavLink, 'nav-nested-item')),
+        href: undefined,
```

Comment:
Nit: maybe leave a comment here to describe why need to set href `undefined` explicitly.

Reply:
Sure, added.